.golangci.yaml

```diff
@@ -0,0 +1,35 @@
+run:
+  timeout: 10m
+  allow-parallel-runners: true
+
+linters:
+  enable:
+    - errname
+    - gofmt
```

Comment:
This linter is now failing on windows, but passed on linux an macos. https://github.com/janos/singleflight/actions/runs/7943839023/job/21688910906#step:5:26.

Reply:
i am not sure why that is, so i commented out `gofmt` for now.